php/tests/force_c_ext.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+// We have to test this because the command-line argument will fail silently
+// if the extension could not be loaded:
+//    php -dextension=ext/google/protobuf/modules/protouf.so
+if (!extension_loaded("protobuf")) {
+    throw new Exception("Protobuf extension not loaded");
+}
+
+spl_autoload_register(function($class) {
+    if (strpos($class, "Google\\Protobuf") === 0) {
+        throw new Exception("When using the C extension, we should not load runtime class: " . $class);
```

Comment:
well known types are bundled in c extension?

Reply:
Yes, they are as of https://github.com/protocolbuffers/protobuf/pull/7944